text/0034-telemetry-policies.md

```diff
@@ -108,29 +128,48 @@ This property allows us to gather essential information to better understand on
+| system.distribution    | Distribution on which MeiliSearch is launched          | `Arch Linux`      |
+| system.kernel_version  | Kernel version on which MeiliSearch is launched        | `5.14.10-arch1-1` |
+| system.cores           | Number of cores                                        | `24`              |
+| system.ram_size        | Total RAM capacity. Expressed in `KB`                  | `33604210`        |
+| system.disk_size       | Total capacity of the largest disk. Expressed in `Bytes`| `336042103`       |
+| system.server_provider | Users can tell us on which provider MeiliSearch is hosted by filling the `MEILI_SERVER_PROVIDER` env var. This is also filled by our providers deploy scripts. e.g. [GCP cloud-config.yaml](https://github.com/meilisearch/cloud-scripts/blob/56a7c2630c1a508e5ad0c0ba1d8cfeb8d2fa9ae0/scripts/providers/gcp/cloud-config.yaml#L33) | `gcp` |
 
 ##### MeiliSearch Configuration `context` and `infos`
 
 | Property name | Description | Example |
 |---------------|-------------|---------|
 | context.app.version  | MeiliSearch version number.  Sent in a `context` object instead of `properties` to match Amplitude requirement | 0.23.0 |
-| infos.env | MeiliSearch env. `production` / `development` | `production` |
-| infos.has_snapshot| `true` if snapshots are activated, otherwise `false` | `true` |
+| infos.env | Value of `--env`/`MEILI_ENV` | `production` |
+| infos.db_path | `true` if `--db-path`/`MEILI_DB_PATH` is specified, otherwise `false` | `true`|
+| infos.import_dump | `true` if `--import-dump` is specified, otherwise `false` | `true` |
+| infos.dumps_dir | `true` if `--dumps-dir`/`MEILI_DUMPS_DIR` is specified, otherwise `false` | `true` |
+| infos.ignore_missing_dump | `true` if `--ignore-missing-dump` is specified to true, otherwise `false` | `true` |
+| infos.ignore_dump_if_db_exists | `true` if `--ignore-dump-if-db-exists` is specified to true, otherwise `false` | `true` |
+| infos.import_snapshot | `true` if `--import-snapshot` is specified, otherwise `false` | `true` |
+| infos.schedule_snapshot | `true` if `--schedule_snapshot`/`MEILI_SCHEDULE_SNAPSHOT` is specified to true, otherwise `false` | `true` |
+| infos.snapshot_dir | `true` if `--snapshot-dir`/`MEILI_SNAPSHOT_DIR` is specified to true, otherwise `false` | `true` |
+| infos.snapshot_interval_sec | Value of `--snapshot_interval_sec`/`MEILI_SNAPSHOT_INTERVAL_SEC` in seconds | `86400` |
+| infos.ignore_missing_snapshot | `true` if `--ignore_missing_snapshot` is specified to true, otherwise `false` | `true` |
+| infos.ignore_snapshot_if_db_exists | `true` if `--ignore_snapshot_if_db_exists` is specified to true, otherwise `false` | `true` |
```

Comment:
```suggestion
| infos.ignore_snapshot_if_db_exists | `true` if `--ignore-snapshot-if-db-exists` is specified to true, otherwise `false` | `true` |
```